lib/openstudio-standards/utilities/object_info.rb

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
this method is never used in the codebase and should be deleted, as well as the reference to it in lib_openstudio-standards.rb

Reply:
Can we keep this in? I seem to recall that it was created for debugging purposes. It looks like Jerry created it, and he is out this week.

Reply:
document that it is an OS model object